content/workload-reduction-toolkit/address-workload-issues/communications/switching-to-online-parents-evenings.md

```diff
@@ -22,10 +25,8 @@ title: Switching to online parents' evenings
 
 {/inset-text}
 
----
-
 <div class="govuk-grid-row dfe-width-container">
```

Comment:
I was just thinking, this more complicated code for the boxes - is there any way we could render this, similarly to the feedback boxes you've already done?
this might simplify the code for the individual resources to make them easier to update, but also means if we ever adjusted the boxes we could do it in 1 place and it would copy to everywhere else?